src/deploy/table_social_network.sql

```diff
@@ -0,0 +1,20 @@
+-- Deploy maevsi:table_social_network to pg
+-- requires: schema_public
+
+BEGIN;
+
+CREATE TABLE maevsi.social_network (
+  name TEXT PRIMARY KEY
+);
```

Comment:
Do you see upcoming extensions of this information or would an enumeration better fit this more or less fixed set of elements?
If we expect non-developers, like when event organizers, moderators, ... to manage such entities (social networks), we should create this table, but I think maevsi does not intend to offer social network tweaking.

Reply:
Additional records to this table will happen rarely but I prefer tables like this to enum types.

Reply:
Ok, I want to keep the API interface reasonable, so I converted the table into an enumeration which only appears as a type in the GraphQL API.